packages/checkout-storefront/src/sections/OrderInfo/OrderInfo.tsx

```diff
@@ -1,6 +1,9 @@
 import { Address } from "@/checkout-storefront/components/Address";
 import { OrderFragment } from "@/checkout-storefront/graphql";
 import { useFormattedMessages } from "@/checkout-storefront/hooks/useFormattedMessages";
+import { messages as billingMessages } from "@/checkout-storefront/sections/BillingAddressSection/messages";
+import { messages as contactMessages } from "@/checkout-storefront/sections/Contact/messages";
+import { messages as shippingMessages } from "@/checkout-storefront/sections/ShippingAddressSection/messages";
```

Comment:
Can we change the exported variable names from `messages` to `billingMessages`, ... ?